cli/cmd/sync.go

```diff
@@ -221,6 +221,9 @@ func sanitizeComposeFile(composeFilePath string, targetFilePath string) error {
 		return err
 	}
 
+	// sets version to 2.4 for testing purpose
```

Comment:
Not testing anymore?

```suggestion
```

Reply:
Well, I mean testing the execution of the tests. I added there to explain why we are setting that version there. Maybe it's not clear enough and it could be deleted